src/app/legacy/components/RadioSchedule/StartTime/__snapshots__/index.test.jsx.snap

```diff
@@ -32,7 +32,6 @@ exports[`StartTime should render LTR correctly 1`] = `
 
 @media screen and (forced-colors: active) {
   .emotion-2>svg {
-    padding-right: 0.25rem;
```

Comment:
These snapshot changes are actually correct. There shouldn't have been padding applied here:

This is the component:
```
const StyledClock = styled.span`
  display: flex;
  align-items: center;
  ${({ dir }) =>
    dir === 'ltr'
      ? `padding-right: ${GEL_SPACING_HLF};`
      : `padding-left: ${GEL_SPACING_HLF};`}
  > svg {
    color: ${props => props.theme.palette.RHINO};
    margin: 0;
    overflow: visible;
    @media screen and (forced-colors: active) {
      fill: canvasText;
    }
  }
`;
```
Note that the `svg` shouldn't have padding in it, it should only be in the parent, which it is too but for some reason leaked into the `svg` element.

Reply:
How come the snapshots have changed - I was expecting to see a change to an index file which caused it, but is it actually because of one of the emotion libraries?

Reply:
Yea possibly https://github.com/emotion-js/emotion/releases/tag/%40emotion%2Fstyled%4011.11.0 or https://github.com/emotion-js/emotion/releases/tag/%40emotion%2Fstyled%4011.10.8

Looks like a change to the parser. Could have resulted in more strict/accurate styles being generated.

Reply:
Does this maybe need testing manually to see if it's made anything look incorrect?

Reply:
Would the visual snapshots catch this?

Reply:
Not visual changes were found in Chromatic for where this was used, so I don't think there is any visual regression removing this.